opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -172,11 +179,10 @@ impl<R: RuntimeChannel> LogProcessor for BatchLogProcessor<R> {
         )));
 
         // TODO - Implement throttling to prevent error flooding when the queue is full or closed.
-        if let Err(err) = result {
-            otel_error!(
-                name: "BatchLogProcessor.Export.Error",
-                error = format!("{}", err)
-            );
+        if result.is_err() {
```

Comment:
for the very first time, lets also emit an internal log informing the user about data drop.

Reply:
Added this to each processor:

https://github.com/open-telemetry/opentelemetry-rust/blob/b626d7f5b00c336b0d1c263df9e43521269d8bee/opentelemetry-sdk/src/logs/log_processor.rs#L185-L188